oex_plugin/openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -0,0 +1,48 @@
+"""
+Registration related views.
+"""
+
+from django.conf import settings
+from django.core.exceptions import NON_FIELD_ERRORS, PermissionDenied
+from django.core.validators import ValidationError
+from django.http import HttpResponseForbidden
+from django.utils.translation import gettext as _
+
+
+from common.djangoapps.student.helpers import AccountValidationError
+from onboarding.models import UserExtendedProfile
+from openedx.core.djangoapps.user_authn.views.register import create_account_with_params
+
+
+def _create_account(original_func, self, request, data):
+    """
+    Extending standard registration by creating an instance UserExtendedProfile.
+    """
+    response, user = None, None
+    try:
+        user = create_account_with_params(request, data)
+        # Start of override for custom registration
+        UserExtendedProfile.objects.create(user=user)
```

Comment:
could you point me to where a user is linked to the organization?

do we have such a connection?

Reply:
`philu-rg-openedx-plugin/onboarding/views.py`:
```
def save_organization(request):
    ...
    user_extended_profile = request.user.extended_profile
    ...
    user_extended_profile.organization = organization
```